src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -213,11 +213,11 @@ public void withRMarkdown(String progressCaption, String userAction,
       deps.add(Dependency.cranPackage("htmltools", "0.3.5"));
       deps.add(Dependency.cranPackage("caTools", "1.14"));
       deps.add(Dependency.cranPackage("bitops", "1.0-6"));
-      deps.add(Dependency.cranPackage("knitr", "1.14", true));
+      deps.add(Dependency.cranPackage("knitr", "1.14"));
```

Comment:
It's been this way for quite some time (see https://github.com/rstudio/rstudio/commit/539c20f83635d1d16f0b106585d533c62d91bba0), even for older knitr, which suggests to me that this isn't done for a transient reason, although I don't recall the original motivation.